package.json

```diff
@@ -71,12 +66,8 @@
   },
   "devDependencies": {
     "@babel/core": "^7.13.8",
+    "@babel/plugin-proposal-numeric-separator": "^7.14.5",
     "@sentry/webpack-plugin": "^1.15.1",
-    "@storybook/addon-actions": "^6.3.7",
-    "@storybook/addon-console": "^1.2.3",
-    "@storybook/addon-essentials": "^6.3.7",
-    "@storybook/addon-links": "^6.3.7",
-    "@storybook/vue3": "^6.3.7",
```

Comment:
Removing Storybook for now. We barely use it and it causes a lot of bloat. Also causes a lot of version conflicts. Lets see if we can go without it for now and maybe we can re-introduce if we break the component library away from the main app.